### PR TITLE
Added flag for single or comma separated stdin input values (#737)

### PR DESCRIPTION
**Pull request by ariasmn**
There are a couple of things which I don't know if are ok:
- Variable/function naming. Non-native speaker so maybe the spelling is wrong.
- You can input from the stdin and a file (using both the **_-l_** and _**-u**_ flags) at the same time.
- Did not add any tests because I didn't fully understand how these works. I can do some if needed with some guidance.

### CODE REVIEW

**Comment by ariasmn** [comment]
Hey, just wanted to check the current status of the PR and ask if there is anything more I could do to help

File: runner/runner.go [hunk 1]
```diff
@@ -316,8 +316,19 @@ func (r *Runner) prepareInputPaths() {
 }
 
 func (r *Runner) prepareInput() {
-	// check if file has been provided
 	var numHosts int
+	// check if a string input has been provided
+	if r.options.InputTargetHost != "" {
```
Comment (forgedhallpass) [inline runner/runner.go:321]: We should use the logic from `goflags` that automatically splits the input into a slice. See `StringSliceVarP`

Comment (forgedhallpass) [reply runner/runner.go:321]: You could check how it has been done in other projects of ours (e.g. `nuclei` or `dnsx`)

**Review by forgedhallpass — CHANGES_REQUESTED** [review]
Please see my comments.

**Comment by ariasmn** [comment]
Done! Just updated the PR.

Please let me know if everything is ok.

**Review by Ice3man543 — APPROVED** [review]
lgtm

**Review by forgedhallpass — APPROVED** [review]

**Review by ehsandeep — APPROVED** [review]
```console
./httpx -u example.com,google.com -silent

https://google.com
https://example.com
```

**Pull request merged**